resources/user-guides/launch-data-elements.md

```diff
@@ -0,0 +1,80 @@
+# Data Elements
+
+Data elements are the building blocks for your data dictionary and are used to collect, organize, and deliver data across marketing and ad technology.
+
+A data element is a variable where the value can be mapped to data within the Experience Platform Mobile SDK such as visitor ID, device name, user profile, number of launches, et cetera. In Experience Platform Launch you can reference this value by its variable name. This collection of data elements becomes the dictionary of defined data that you can use to build rules for your application, and this dictionary is shared across Experience Platform Launch where it can be used with any extension added to your property.
+
+Use data elements as widely as possible throughout rule creation to consolidate the definition of dynamic data. You define data elements once, and use them in multiple places. The concept of reusable data elements if very powerful and you should use them as a best practice.
+
+Data elements are populated with data when they are processed within the Experience Platform Mobile SDK [Rules Engine](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine). To use data elements, at a high level:
+
+1. Create a data element.
+2. Use the data element when defining a rule.
+3. [Publish](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html) your property, which publishes the rules JSON and make it available for download by your applications.
+
+## Create a data element
+
+Data elements are building blocks for rules. Data elements let you create a data dictionary of commonly used data in the Experience Platform Mobile SDK, regardless of where they originate (shared state, event data) or which extension creates them.
+
+1. From a Property page, open the Data Elements tab, then click Add Data Element.
+
+2. Give a unique, descriptive name to the data element.
+
+3. Select the Extension which generates the data element type. 
+
+   You may only select Extensions which are currently installed in your property.
+
+4. Select the Data Element Type. 
+
+   Add the required configuration parameters. Most data element types do not require any configuration.
+
+5. Click Save.
+
+For example, to create a data element which maps to the Experience Cloud ID, from the Create New Data Element page, select Extension *Mobile Core*, Data Element Type *Experience Cloud ID* and enter Name *ECID*. 
```

Comment:
Can we also give an example where there is configuration needed, e.g. profile key/context data or do you think this example is enough https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/signals/signals-extension-and-rules-engine-integration?